Tasks/RunRemoteShellCommandsSSH/task.json

```diff
@@ -0,0 +1,89 @@
+{
+    "id": "91443475-df55-4874-944b-39253b558790",
+    "name": "RunRemoteShellCommandsSSH",
+    "friendlyName": "Remote Shell Commands",
+    "description": "Run shell commands on a remote machine using SSH.",
+    "helpMarkDown": "",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "demands": [
+     ],
+    "instanceNameFormat": "Run shell commands on remote machine using SSH",
+    "groups": [
+        {
+            "name": "connection",
+            "displayName": "Connection",
+            "isExpanded": true
+        }
+    ],
+    "inputs": [
+        {
+            "name": "hostname",
```

Comment:
Why isn't this a service endpoint of type SSH?  That's what a service endpoint is - credentials with a location.  they are also extensible so it would be easy to add type SSH.  Then customer would have dropdown here (with manage link).  That means you could enter that information once and use it across multiple instances of the task.  That also means if the credential changes you have one place to edit.

This shouldn't go in like this.